extensions/amp-list/0.1/amp-list.js

```diff
@@ -134,6 +134,10 @@ export class AmpList extends AMP.BaseElement {
       }
     }, ActionTrust.HIGH);
 
+    this.registerAction('containerize',
```

Comment:
let's guard the registration around the experiment flag

Reply:
Done